core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/stacks/base/NetworkV4Base.java

```diff
@@ -135,4 +138,31 @@ public MockNetworkV4Parameters getMock() {
     public YarnNetworkV4Parameters getYarn() {
         return yarn;
     }
+
+    public boolean isEmpty() {
+        for (Field declaredField : NetworkV4Base.class.getDeclaredFields()) {
+            declaredField.setAccessible(true);
```

Comment:
is it required? I think these fields are accessible inside this class. Could you please add unit tests for this method?

Reply:
Yeah, will make a refactor. I made this method to simply check whether the given instance is empty (all fields are empty). In this way, the `isEmpty` method shall tell the emptiness even if the number of fields will change in the future.

Reply:
I don't mean the method, I mean the exact line. You set it to accessible in runtime. I think it is unneccesarly.

Reply:
that is true, now it is removed